llvm/lib/Target/AArch64/Utils/AArch64SMEAttributes.h

```diff
@@ -12,6 +12,9 @@
 #include "llvm/IR/Function.h"
 
 namespace llvm {
+namespace RTLIB {
+struct RuntimeLibcallsInfo;
+}
 
 class AArch64TargetLowering;
```

Comment:
Remove the `AArch64TargetLowering` forward decl (and I think the include of `AArch64ISelLowering.h` in the C++ file can be replaced).